rtsp-to-web/build.yaml

```diff
@@ -1,14 +1,13 @@
-# https://developers.home-assistant.io/docs/add-ons/configuration#add-on-dockerfile
+# https://developers.home-assistant.io/docs/add-ons/configuration/#add-on-extended-build
```

Comment:
Can you help me understand more the motivation behind changing the base image from the example? My impression is this adds extra build labels for the community addons, but this isn't a community add-on.

Reply:
Ops, I think now I have a better reasoning to prefer the new one:

```console
❯ docker images
REPOSITORY                          TAG       IMAGE ID       CREATED       SIZE
ghcr.io/home-assistant/amd64-base   3.14      e6325b527bd3   6 hours ago   68.4MB
ghcr.io/hassio-addons/base/amd64    11.0.1    a0e49bd90483   4 weeks ago   25.5MB
```

Reply:
Yes, this is working, great!
```
$ docker pull ghcr.io/felipecrs/armv7-rtsp-to-web:latest
$ docker run -p 8083:8083 ghcr.io/felipecrs/armv7-rtsp-to-web:latest
...
[16:44:29] INFO: Starting discovery service
[16:44:29] INFO: Starting RSTPtoWeb
```

Reply:
I still don't want these labels, maintainer, etc.